### DISCUSSION TITLE
Trainer Agreement comments and notes

### DISCUSSION THREAD

**Discussion by ACharbonneau** [carpentries/trainers#114] [Trainer Practices]
At the May trainers leadership meeting, we decided to assess what changes need to be done to the trainer agreement while we find a committee to work on the update process. We'll use that process to facilitate the updates that we work out here.

[Link to current trainer agreement](https://docs.carpentries.org/topic_folders/instructor_training/duties_agreement.html)

**Comment by brownsarahm** [comment]
Noting some meeting minutes that capture previous concerns from trainers:


[July-August 2020](https://github.com/carpentries/trainers/blob/main/minutes/July-August_2020_Notes.md) 
_note this is multiple meetings in one file_

**Comment by brownsarahm** [comment]
_this is structured for readabilty, but represents rough brainstorming level ideas, not a firm plan yet, all of this is subject to revision and some of it is incomplete_

One thought is that the current "Trainer Agreement" is doing a lot of work. There are two key functions that I'd like to separate out during this year's update:
- stating what is required for a trainer to remain a certified trainer
- providing time estimates to help the core team with planning

These two things are not totally unrelated and so I think the current definition of "units" is useful to carry across both steps and TBD where that definition lives. 


In the actual update process trainers would:
1. Opt for alumni or active (accepting that they understand the minimum requirement for active) 
2. if opting to stay active, provide an estimate of their availability (completely nonbinding) this could include allocation of units to full workshops vs bonus modules vs demos and possibly a breakdown over the year if that would be helpful

We could have the "minimum to be active" be fewer units than what's in the current agreement or keep it at the same level.  If we have a lower active bar, we should think carefully about incentives around people doing above and beyond that (other than those whose time is actually included in a membership.
